packages/@aws-cdk/aws-ecs/README.md

```diff
@@ -352,3 +352,118 @@ rule.addTarget(new targets.EcsTask({
   }]
 }));
 ```
+
```

Comment:
@rix0rrr I added a quick section in the readme. Let me know if you think this is uneccessary.